phones/models.py

```diff
@@ -194,7 +194,8 @@ class RelayNumber(models.Model):
 
     @property
     def remaining_minutes(self):
-        return floor(self.remaining_seconds / 60)
+        minutes_called = floor(self.remaining_seconds / 60)
+        return minutes_called if minutes_called > 0 else 0
```

Comment:
I'm used to seeing this using `max`:
```suggestion
        return floor(max(self.remaining_seconds, 0) / 60))
```

but I think explicit is better than implicit

Reply:
completed 8e485f0